<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 236, 236);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="answer">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#answer {
	width: 360px;
	height: 36px;
	border-radius: 9px;
	background-color: 'white';
}

#answer:hover {
	background-color: #F8F8FF;
}</string>
      </property>
      <property name="text">
       <string>Answer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QLabel" name="patient_waiting_label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-left:26%</string>
      </property>
      <property name="text">
       <string>Patients waiting:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QLabel" name="logo">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-left: 16%</string>
      </property>
      <property name="text">
       <string>SeeComm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignTop">
     <widget class="QLabel" name="patient_waiting_number">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-top: '6px';</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
